tests/sentry/mediators/sentry_apps/test_updater.py

```diff
@@ -54,6 +55,19 @@ def test_doesnt_update_app_with_invalid_event_permissions(self):
         with self.assertRaises(APIError):
             updater.call()
 
+    def test_updates_service_hook_events(self):
+        sentry_app = self.create_sentry_app(
+            name='sentry',
+            organization=self.org,
+            scopes=('project:read', 'event:read',),
```

Comment:
Maybe we can explicitly pass a list of events that is _different_ than `('issue',)` here to ensure it's actually changing the value.